### ISSUE TITLE
DCT ICO Balance transfer to Decent GO Wallet showing as 0 instead of true balance.

### ISSUE THREAD

**Issue by bss-** [DECENTfoundation/DECENT-Network#216]
I followed the ICO reclaim guide on 11 July. 

My balance shown at the end of this process was **1375** DCT.  (Screenshot: DCT ICO Balance.png)
But when I login to my wallet on decentgo it shows **0** DCT.  (Screenshot: DCT Wallet Balance.png)

Please fix to show the correct balance of 1375 so that I can withdraw my DCT.

![dct ico balance](https://user-images.githubusercontent.com/8261177/28886276-cb7e32d6-77b0-11e7-9fe9-705bc64303c7.png)

![dct wallet balance](https://user-images.githubusercontent.com/8261177/28886326-fec8dc72-77b0-11e7-8629-9caa8d5e47b0.png)

**Comment by Texxer** [comment]
I suppose you're at the wrong place here. This board is made for developers.

On https://www.decentgo.com/ you'll find a link on the bottom to submit a ticket for your problem.

**Comment by ejossev** [comment]
Anyway, the funds have been transferred to bittrex:
```
unlocked >>> get_account_history u53f64dd6d77f93acce111d1ed2da3629 5
get_account_history u53f64dd6d77f93acce111d1ed2da3629 5
[{
    "memo": "",
    "description": "Transfer 673.88500000 DCT from u53f64dd6d77f93acce111d1ed2da3629 to bittrex -- could not decrypt memo   (Fee: 0.00500000 DCT)",
    "op": {
      "id": "1.7.35929",
      "op": [
        0,{
          "fee": {
            "amount": 500000,
            "asset_id": "1.3.0"
          },
          "from": "1.2.3207",
          "to": "1.2.22",
          "amount": {
            "amount": "67388500000",
            "asset_id": "1.3.0"
          },
          "memo": {
            "from": "DCT51HHYzQa6dhFr351LxTKvPGv51uT1FLrWUJhjGJiprNvfMN5wv",
            "to": "DCT6nEbRi4pbp4s62m1C73iwmHFVuCzvpZFBP7AGTdJPYNXj6Hf58",
            "nonce": "384470541269107",
            "message": "2177c6f762b250ff1e446ee8fa84ed06679311a0b0293668ebdeea35f658b32d"
          },
          "extensions": []
        }
      ],
      "result": [
        0,{}
      ],
      "block_num": 573307,
      "trx_in_block": 0,
      "op_in_trx": 0,
      "virtual_op": 36930
    }
  },{
    "memo": "",
    "description": "Transfer 700 DCT from u53f64dd6d77f93acce111d1ed2da3629 to bittrex -- could not decrypt memo   (Fee: 0.00500000 DCT)",
    "op": {
      "id": "1.7.35921",
      "op": [
        0,{
          "fee": {
            "amount": 500000,
            "asset_id": "1.3.0"
          },
          "from": "1.2.3207",
          "to": "1.2.22",
          "amount": {
            "amount": "70000000000",
            "asset_id": "1.3.0"
          },
          "memo": {
            "from": "DCT51HHYzQa6dhFr351LxTKvPGv51uT1FLrWUJhjGJiprNvfMN5wv",
            "to": "DCT6nEbRi4pbp4s62m1C73iwmHFVuCzvpZFBP7AGTdJPYNXj6Hf58",
            "nonce": "384470472508530",
            "message": "eb57acc60953cdde54f40c4937241769e9b308a3369e09d4016e98896a8b8b18"
          },
          "extensions": []
        }
      ],
      "result": [
        0,{}
      ],
      "block_num": 573253,
      "trx_in_block": 0,
      "op_in_trx": 0,
      "virtual_op": 36911
    }
  },{
    "memo": "",
    "description": "Transfer 1.10000000 DCT from u53f64dd6d77f93acce111d1ed2da3629 to bittrex -- could not decrypt memo   (Fee: 0.00500000 DCT)",
    "op": {
      "id": "1.7.35903",
      "op": [
        0,{
          "fee": {
            "amount": 500000,
            "asset_id": "1.3.0"
          },
          "from": "1.2.3207",
          "to": "1.2.22",
          "amount": {
            "amount": 110000000,
            "asset_id": "1.3.0"
          },
          "memo": {
            "from": "DCT51HHYzQa6dhFr351LxTKvPGv51uT1FLrWUJhjGJiprNvfMN5wv",
            "to": "DCT6nEbRi4pbp4s62m1C73iwmHFVuCzvpZFBP7AGTdJPYNXj6Hf58",
            "nonce": "384470337582193",
            "message": "60f75075940dd652b7ae71b63f2010bce36a4bb12a138beeebc6fb34a27eabf0"
          },
          "extensions": []
        }
      ],
      "result": [
        0,{}
      ],
      "block_num": 573147,
      "trx_in_block": 0,
      "op_in_trx": 0,
      "virtual_op": 36874
    }
  },{
    "memo": "",
    "description": "Transfer 1375 DCT from decent2 to u53f64dd6d77f93acce111d1ed2da3629   (Fee: 0.00500000 DCT)",
    "op": {
      "id": "1.7.35070",
      "op": [
        0,{
          "fee": {
            "amount": 500000,
            "asset_id": "1.3.0"
          },
          "from": "1.2.16",
          "to": "1.2.3207",
          "amount": {
            "amount": "137500000000",
            "asset_id": "1.3.0"
          },
          "extensions": []
        }
      ],
      "result": [
        0,{}
      ],
      "block_num": 560695,
      "trx_in_block": 0,
      "op_in_trx": 0,
      "virtual_op": 35498
    }
  },{
```

**Comment by bss-** [comment]
I submitted a support ticket on decent.ladesk.com on **13 July**. 
But after several emails back and forth no progress was made and my DCT had not been received as of **2 August**.
So I decided to go direct to the developers by submitting this Github issue. 
It seems to have had the desired effect as the very next day, **3 August**, my DCT has been transferred to me.
(Screenshot below: DCT on decentgo.png).

Also, there was a comment on the Decent Blog about this (https://decent.ch/en/hot-summer-update-for-decent-go/), that implies that the developers were working on this bug:

> A handful of users have been experiencing 0 balance issues, even though their accounts are showing a healthy DCT balance. This update should address these issues, but perhaps more importantly, moving forward if these errors persist or new issues arise the system will return error messages to enable our devs to locate the source of the issues and take appropriate action to get you up and running much faster.

As noted above I've received and transferred my DCT out so the issue is resolved for me. I wonder whether the bug is now fixed.

![dct on decentgo](https://user-images.githubusercontent.com/8261177/29005177-5ad08c6c-7acd-11e7-89b9-a5adcc406de0.PNG)
